docs/dashboards/overview-dashboard.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[overview-dashboard]]
+= Overview dashboard
+
+The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation.
+
+image::images/overview-pg.png[Overview dashboard]
+
+[discrete]
+== Live feed
+
+The live feed on the Overview dashboard helps you quickly access recently created cases, favorited Timelines, and the latest {elastic-sec} news.
+
+TIP: The *Security news* section provides the latest {elastic-sec} news to help you stay informed of new developments, learn about {elastic-sec} features, and more.
+
+image::images/live-feed-ov-page.png[Overview dashboard with live feed section highlighted]
+
+[discrete]
+== Histograms
+
+Time-based histograms show you the number of detections, alerts, and events that have occurred within the selected time range. To focus on a particular time, click and drag to select a time range, or choose a preset value. The *Stack by* dropdown lets you select which field is used to organize the data. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display alert counts by rule name within the specified time frame.
```

Comment:
Per UX leads, we're supposed to use `menu` instead of `dropdown` (I personally like dropdown, but that was the consensus 😉 ) 
```suggestion
Time-based histograms show the number of detections, alerts, and events that have occurred within the selected time range. To focus on a particular time, click and drag to select a time range, or choose a preset value. The *Stack by* menu lets you select which field is used to organize the data. For example, in the Detection alert trend histogram, stack by `kibana.alert.rule.name` to display alert counts by rule name within the specified time frame.
```